### DISCUSSION TITLE
Support for `grep --binary-files=binary` and `rg --binary`

### DISCUSSION THREAD

**Discussion by nick4f42** [minad/consult#1157] [Ideas]
Recently, I was confused why `consult-ripgrep` wasn't finding a string that I knew was in a certain file. It turned out that there was an extraneous null byte in the file, causing the file to be skipped. It would be nice if consult could display `file.txt: binary file matches` and link to the file in this case.

**Comment by minad** [comment]
Good point. Patches welcome which add this.
